pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1957,6 +1989,17 @@ type RestoreList struct {
 	Items []Restore `json:"items"`
 }
 
+// RestoreType represents the restore type.
+// +k8s:openapi-gen=true
+type RestoreType string
```

Comment:
Define a new term to make it different to BackupType, maybe RestoreMode is better.

Reply:
ok